spec/jekyll_seo_tag_spec.rb

```diff
@@ -237,11 +237,49 @@
     end
 
     context "with absolute site.logo" do
-      let(:site) { make_site("logo" => "http://cdn.example.invalid/logo.png", "url" => "http://example.invalid") }
+      let(:site) { make_site("logo" => "http://cdn.example.invalid/logo.png", "url" => "http://example.invalid", "author" => "Mr. Foo") }
```

Comment:
I'll defer this one. I don't have the experience with tests to make a good judgement here. Just wafting around code smell. 💨